### ISSUE TITLE
Improve --html reports in order to display the full payloads content

### ISSUE THREAD

**Issue by adaniloff** [Codeception/Codeception#5869]
#### What are you trying to achieve?

I'm trying to use the CLI in order to generate an html report with the `--html` option.
Unfortunately, I've noticed that codeception truncate automatically the result in the generated file, if it exceed 200 characters.

See below:
```html
<!-- If my line exceed 200 chars, it will be truncated -->
<!-- Let's image that our line is like that -->
<td class="stepName ">&nbsp;&nbsp;&nbsp;&nbsp;I see response contains json <span style="color: #732E81">{&quot;total&quot;:1015.63,&quot;content&quot;:[{&quot;price&quot;:1000,&quot;line_index&quot;:1,&quot;product&quot;:{&quot;id&quot;:259},&quot;is_discount&quot;:false,&quot;quantity&quot;:1},{&quot;price&quot;:1000,&quot;line_index&quot;:2,&quot;product&quot;:{&quot;id&quot;:260},&quot;is_discount&quot;:false,&quot;quantity&quot;:1},{&quot;price&quot;:11,&quot;line_index&quot;:3,&quot;product&quot;:{&quot;id&quot;:261},&quot;is_discount&quot;:false,&quot;quantity&quot;:1},{&quot;price&quot;:-200,&quot;product&quot;:{&quot;id&quot;:259},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:252,&quot;quantity&quot;:1},{&quot;price&quot;:-100,&quot;product&quot;:{&quot;id&quot;:260},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:250,&quot;quantity&quot;:1},{&quot;price&quot;:-10,&quot;product&quot;:{&quot;id&quot;:261},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:251,&quot;quantity&quot;:1},{&quot;price&quot;:-240,&quot;product&quot;:{&quot;id&quot;:259},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:253,&quot;quantity&quot;:1},{&quot;price&quot;:-270,&quot;product&quot;:{&quot;id&quot;:260},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:253,&quot;quantity&quot;:1},{&quot;price&quot;:-0.3,&quot;product&quot;:{&quot;id&quot;:261},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:253,&quot;quantity&quot;:1},{&quot;price&quot;:-56,&quot;product&quot;:{&quot;id&quot;:259},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:254,&quot;quantity&quot;:1},{&quot;price&quot;:-56,&quot;product&quot;:{&quot;id&quot;:259},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:255,&quot;quantity&quot;:1},{&quot;price&quot;:-63,&quot;product&quot;:{&quot;id&quot;:260},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:255,&quot;quantity&quot;:1},{&quot;price&quot;:-0.07,&quot;product&quot;:{&quot;id&quot;:261},&quot;line_index&quot;:null,&quot;is_discount&quot;:true,&quot;discount_id&quot;:255,&quot;quantity&quot;:1}],&quot;discounts&quot;:[{&quot;dispatch&quot;:[{&quot;line_index&quot;:1}],&quot;is_basket_impacted&quot;:false,&quot;is_auto&quot;:false,&quot;discount&quot;:{&quot;id&quot;:254},&quot;amount&quot;:56},{&quot;dispatch&quot;:[{&quot;line_index&quot;:1},{&quot;line_index&quot;:2},{&quot;line_index&quot;:3}],&quot;is_basket_impacted&quot;:true,&quot;is_auto&quot;:false,&quot;discount&quot;:{&quot;id&quot;:255},&quot;amount&quot;:119.07},{&quot;dispatch&quot;:[{&quot;line_index&quot;:1}],&quot;is_basket_impacted&quot;:false,&quot;is_auto&quot;:true,&quot;discount&quot;:{&quot;id&quot;:252},&quot;amount&quot;:200},{&quot;dispatch&quot;:[{&quot;line_index&quot;:2}],&quot;is_basket_impacted&quot;:false,&quot;is_auto&quot;:true,&quot;discount&quot;:{&quot;id&quot;:250},&quot;amount&quot;:100},{&quot;dispatch&quot;:[{&quot;line_index&quot;:3}],&quot;is_basket_impacted&quot;:false,&quot;is_auto&quot;:true,&quot;discount&quot;:{&quot;id&quot;:251},&quot;amount&quot;:10},{&quot;dispatch&quot;:[{&quot;line_index&quot;:1},{&quot;line_index&quot;:2},{&quot;line_index&quot;:3}],&quot;is_basket_impacted&quot;:true,&quot;is_auto&quot;:true,&quot;discount&quot;:{&quot;id&quot;:253},&quot;amount&quot;:510.3}]}</span></td>

<!-- Then it will be truncated like that -->
<td class="stepName ">&nbsp;&nbsp;&nbsp;&nbsp;I see response contains json <span style="color: #732E81">{&quot;total&quot;:1015.63,& ...}</span></td>

<!-- In this example, I truncated way before 200 chars in order to make it easier to read -->
```

#### What do you get instead?

Truncated output that I can't exploit.

### Details

I figured out that the method `vendor/codeception/codeception/src/Codeception/Step.php:getHtml` calls the method `getHumanizedArguments` without any argument.

Here is the latter: 
```php
<?php
    public function getHumanizedArguments($maxLength = 200)
    {
        return $this->getArgumentsAsString($maxLength);
    }
```

If no argument is passed, it uses a maxLength of 200.

So by changing the line `vendor/codeception/codeception/src/Codeception/Step.php:226` like below, it fixes my issues.

```php
<?php
# changing $this->getHumanizedArguments() for $this->getHumanizedArguments(0)
        return sprintf('%s %s <span style="color: %s">%s</span>', ucfirst($this->prefix), htmlspecialchars($this->humanize($this->getAction())), $highlightColor, htmlspecialchars($this->getHumanizedArguments(0)));

```

However, I'm not sure that it won't have any side-effect at this point, since I did not looked it up deeply.

**Comment by adaniloff** [comment]
See PR #5870 for a quick fix example (not the best solution IMO).

**Comment by Naktibalda** [comment]
Fixed by #5870
